.detoxrc.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
     },
     'android.emulator': {
       type: 'android.emulator',
-      device: { avdName: 'Pixel_7_Pro_API_34' },
+      device: { avdName: 'Pixel_7_Pro_API_VanillaIceCream' },
```

Comment:
I think you can just rename your sim locally instead of making everyone else rename theirs